reconcile/gitlab_owners.py

```diff
@@ -64,10 +75,10 @@ def get_change_owners_map(self):
             closest_reviewers = closest_owners["reviewers"]
 
             change_owners_map[path] = {
-                "approvers": path_approvers,
-                "reviewers": path_reviewers,
-                "closest_approvers": closest_approvers,
-                "closest_reviewers": closest_reviewers,
+                "approvers": self.expand_groups(path_approvers),
+                "reviewers": self.expand_groups(path_reviewers),
+                "closest_approvers": self.expand_groups(closest_approvers),
+                "closest_reviewers": self.expand_groups(closest_reviewers),
```

Comment:
Something is wrong here; `expand_groups` always returns `None`; therefore, you assign `None` to all dictionary fields.
Please change `expand_groups` to return the updated owners list.

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4906/commits/929535687bb0a3953c216f4f2748bd7187bf9da2